pages/index.js

```diff
@@ -244,7 +244,7 @@ class Index extends React.Component {
       body: JSON.stringify({
         query:fetchMessages,
         variables: this.getLastReceivedVars()
-      })
+      }).throw("ERROR")
```

Comment:
Consider providing a meaningful error message instead of just "ERROR" for better debugging and user experience.